mods/d2k/rules/structures.yaml

```diff
@@ -599,7 +606,7 @@ starport:
 	Valued:
 		Cost: 1500
 	Building:
-		Footprint: xxx x=x =x=
+		Footprint: xxx xxx =x=
```

Comment:
This breaks Atreides05 as the smuggler frigate can't land anymore:
![frigateblocked](https://user-images.githubusercontent.com/7704140/27856612-e889d89c-616e-11e7-9986-0ed5f6d2491f.png)